From c239d10fdc91a952ace2d709e445bb049821ae68 Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Sat, 22 Jun 2024 01:35:08 +0200
Subject: [PATCH] Fix lint for baseheader.h (#10)

This previously was overlooked but it is caught by `arc lint`

diff --git a/src/primitives/baseheader.h b/src/primitives/baseheader.h
index 6b62d3d4962024505da01626919507717904a5f2..60b7db41a0a65486f091d2e7527a3561405188ef 100644
--- a/src/primitives/baseheader.h
+++ b/src/primitives/baseheader.h
@@ -2,8 +2,8 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-#ifndef BITCOIN_PRIMITIVES_BASE_HEADER_H
-#define BITCOIN_PRIMITIVES_BASE_HEADER_H
+#ifndef BITCOIN_PRIMITIVES_BASEHEADER_H
+#define BITCOIN_PRIMITIVES_BASEHEADER_H
 
 #include <primitives/blockhash.h>
 #include <serialize.h>
@@ -66,4 +66,4 @@ public:
     int64_t GetBlockTime() const { return (int64_t)nTime; }
 };
 
-#endif // BITCOIN_PRIMITIVES_BASE_HEADER_H
+#endif // BITCOIN_PRIMITIVES_BASEHEADER_H
